### DISCUSSION TITLE
Repo for charge_period_card

### DISCUSSION THREAD

**Discussion by canton7** [nathanmarlor/foxess_modbus#119] [General]
Hey @nathanmarlor 

I'm finally working on part 3 of setting charge periods, which is a custom card.

My understanding is that it needs to live in a separate repository for HACS to be happy with it. If you don't mind, I think it would make sense for you to host this repo, as then users can see that they two are related and are by the same authors? If you agree, please create a repo called foxess_modbus_charge_period_card, and I'll push up what I've got so far.

Thanks!

**Comment by iancg** [comment]
Cheeky ask from me, but would you mind sharing a screen shot of it please @canton7 .  Thanks.

**Reply by canton7** [reply]
I mean, I've been working on it for all of a few hours at this point! It's just a clone of the foxess app's UI, but in HA's styling: two toggles and two time inputs per period, with reset/save buttons at the bottom (which don't yet do anything yet...).

**Reply by canton7** [reply]
I'll grab a screenshot next time I've got a development environment running, but don't expect great things just yet, heh

**Comment by nathanmarlor** [comment]
> Hey @nathanmarlor
> 
> I'm finally working on part 3 of setting charge periods, which is a custom card.
> 
> My understanding is that it needs to live in a separate repository for HACS to be happy with it. If you don't mind, I think it would make sense for you to host this repo, as then users can see that they two are related and are by the same authors? If you agree, please create a repo called foxess_modbus_charge_period_card, and I'll push up what I've got so far.
> 
> Thanks!

Done! :rocket:

**Reply by canton7** [reply]
Thank you!
